README.md

```diff
@@ -74,3 +72,17 @@ C:\devops\DevOps-Course-Starter>poetry run flask run
 
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+Testing 
+The application NEEDS a trello board called ToDo case sensitive 
```

Comment:
(Worth noting you've actually provided an alternative if the user provides a board id, so this isn't quite true!)